### PR TITLE
Faq footer

### PR DESCRIPTION
**Pull request by chottomatta**
Finished FAQ and Footer parts of the project

### CODE REVIEW

File: src/components/footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                0850-390-7444
+              </a>
+            </p>
+
+            <ul className={styles.footerUL}>
+              {footerData &&
+                footerData.map((data, id) => (
+                  <li key={id} className={styles.footerList}>
+                    <a
+                      href="http://localhost:3000/"
```
Comment (barisbabahan) [inline src/components/footer/Footer.js:24]: we can redirect to "/" because in prod it will fail if we click it

File: src/data/faqData.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  {
+    question: "What is Netflix?",
+    answer:
+      "Netflix is a streaming service that offers a wide variety of" +
+      "award-winning TV shows, movies, anime, documentaries, and more on thousands of" +
+      " internet-connected devices." +
+      "\n" +
+      "You can watch as much as you want, whenever you want " +
+      "without a single commercial – all for one low monthly price. There's always something" +
+      " new to discover and new TV shows and movies are added every week!",
```
Comment (barisbabahan) [inline src/data/faqData.js:11]: why this is not just a line string?

File: src/components/faq/FaqDetails.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+              <path d="M10.5 9.3L1.8 0.5 0.5 1.8 9.3 10.5 0.5 19.3 1.8 20.5 10.5 11.8 19.3 20.5 20.5 19.3 11.8 10.5 20.5 1.8 19.3 0.5 10.5 9.3Z"></path>
+            </svg>
+          </button>
+          {isOpen === true ? (
+            <div className={styles.answerDiv}>
+              <span className={styles.answer}>{answer}</span>
+            </div>
+          ) : (
+            ""
+          )}
```
Comment (barisbabahan) [inline src/components/faq/FaqDetails.js:38]:
```
  {isOpen && (
            <div className={styles.answerDiv}>
              <span className={styles.answer}>{answer}</span>
            </div>
          )
}
```

will be better

**Review by barisbabahan — CHANGES_REQUESTED** [review]
Well done you are doing really good!!

Note: as you see on following image the footer items is not centered can you please make them center
local

File: src/data/faqData.js (hunk 2 above)
Comment (chottomatta) [reply src/data/faqData.js:11]: It was looking bad so i made it multiple lines

**Pull request merged**